test/integration/acceptance/fake_peer_example_fixture.hpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_FAKE_PEER_EXAMPLE_FIXTURE_HPP
```

Comment:
This looks like a not directly related change, but I encourage it with a minor exception: let it be called something other than *example*!